app/src/main/java/com/example/bigowlapp/model/Attendance.java

```diff
@@ -0,0 +1,77 @@
+package com.example.bigowlapp.model;
+
+import com.google.firebase.Timestamp;
+import com.google.firebase.firestore.Exclude;
+
+public class Attendance {
+    public enum LocatedStatus {
+        NOT_DETECTED,
+        CORRECT_LOCATION,
+        WRONG_LOCATION
+    }
+
+    private boolean authenticated = false;
+    private boolean authAttemptedMethod1 = false;  // Authentication with first method
+    private boolean authAttemptedMethod2 = false;  // Authentication with second method
```

Comment:
This comment is not requesting a change; I'm only going to give my feedback concerning these lines.

I know this is an issue concerning @Landry333, but I think we should only use a single method of authentication. Also, the boolean called "authenticated" should be "isAuthenticated" as that is a much more precise name and it indirectly states that this field is a boolean.

Reply:
I agree with these points and will discuss with @Landry333